.rspec

```diff
@@ -0,0 +1,3 @@
+--require spec_helper
+--require spec_game
+--require spec_user
```

Comment:
**[REQUIRED]** Use `require_relative` for importing files since it's the Ruby convention.